modules_basic/index.js

```diff
@@ -12,6 +12,8 @@ module.exports = {
   "invite.js":  require('./invite.js'),
   "like.js":  require('./like.js'),
   "markdown.js":  require('./markdown.js'),
+  "message-author.js":  require('./message-author.js'),
+  "message-backlinks.js":  require('./message-backlinks.js'),
   "message-link.js":  require('./message-link.js'),
   "message-name.js":  require('./message-name.js'),
   "message.js":  require('./message.js'),
```

Comment:
would prefer message to be above the message partials ... 
notice in this is starting to feel like it could like a folder....